### ISSUE TITLE
username - student_id??

### ISSUE THREAD

**Issue by Fa8a6** [sreilly1/Final-Year-SE-Project#32]
Hey guys, 

I've just found out that we have two weird columns on the DB, these are:

> - Column: "username" in table: "users"
> - Column: "student_id" in table: "phd_students"

I believe that both are same thing as the username of a PhD student is same as student_id which i guess is student number. Therefore, I guess we should get rid of the student_id column in table **"phd_students"**.

Please let me know if I'm right so I make changes on my stuffs with regard to this :+1: 
Fahad

**Comment by sreilly1** [comment]
I had made the 'username' field as logging with an email may be a bit of a nuisance. But I do not know by heart how to change it so that 'username'is used for logging in with Laravel's built in login functionality, so maybe the 'username' field should go  completely? 

The reason for the student_id column is so  that we have a link back to  the 'users' table so  we can view information about that PHD student that is common to  all other types of users, for example email address or phone number.

**Comment by Fa8a6** [comment]
I agree that there is no need for 'username' in terms of logging in, but even staff members and co-ordinator has unique number that is similar to student number phd students have, I'd say that we take off 'student_id' and leave their student_id value in the 'username' field instead. I hope you got what I mean otherwise I can explain to you in our upcoming meeting.
